migrations.md

```diff
@@ -140,6 +140,9 @@ public function up(): void
 Sometimes a migration might be meant to support a feature that is not yet active and you do not want it to run yet. In this case you may define a `shouldRun` method on the migration. If the `shouldRun` method returns `false`, the migration will be skipped:
 
 ```php
+use App\Models\Flights;
```

Comment:
According to https://github.com/laravel/docs/pull/10186, we prefer having PHP opening tags in the code sample blocks:
```suggestion
```php
<?php

use App\Models\Flights;
```